README.md

```diff
@@ -1,2 +1,38 @@
 # django-multiple-forms
 An expansion of django's FormView to handle multiple form submissions
+
+## How To Use
+#### Install the dependency
+```
+pip install django-mulitform-views
+```
+or add `django-multiform-views` to your `requirements.in` file, then run
+```
+pip compile
+pip install -r requirements.txt
+```
+
+#### Usage
+Have your view inherit from ```FormsView```.
+
+Append `multiform_key` to `helper.layout` in the form
+
+Add the fields for `template_name`, `success_url`, etc as you would normally for `FormView`
```

Comment:
I would remove this line and add something like "all other class properties and methods behave identically to those in Django's FormView" at the end